src/EnergyPlus/DataSurfaces.hh

```diff
@@ -1122,7 +1109,7 @@ namespace DataSurfaces {
         WindowShadingControlData()
```

Comment:
Can eliminate this constructor by adding default assignments to the declaration.